examples/todo/src/main.rs

```diff
@@ -1,10 +1,10 @@
-#![feature(plugin, decl_macro, custom_derive, const_fn)]
+#![feature(plugin, decl_macro, use_extern_macros, custom_derive, const_fn)]
 #![plugin(rocket_codegen)]
 
 extern crate rocket;
-#[macro_use] extern crate diesel;
 #[macro_use] extern crate serde_derive;
 extern crate rocket_contrib;
+#[macro_use] extern crate diesel;
```

Comment:
See the above comment about reexporting Diesel. I still needed to import it here to use its macros.